README.md

```diff
@@ -192,9 +192,15 @@ To deploy the service in a production environment, follow these steps:
 
 2. **Configure Environment Variables**
    - Create a `.env` file in the deployment directory.
-   - Add your OpenAI API key like this:
+   - Obtain all the relevant secrets for the deployment:
+      - `OPENAI_API_KEY`: ...
```

Comment:
@pauljohanneskraft please add some documentation for all the secrets here; some of them are not easily identifiable and we should add small instructions here how to obtain them.